tensorflow/tools/ci_build/builds/docker_test.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
```

Comment:
The main goal of this file (docker_test.sh) is to automate the docker image release process. I have now made further changes to use virtualenv.